docs/kafka/metrics-monitoring-kafka/README.adoc

```diff
@@ -86,27 +86,57 @@ As a developer or administrator, you can view metrics in {product-kafka} to visu
+
+* `kafka_topic:kafka_topic_partitions:count`: Number of topics in the cluster. This does not include internal Kafka topics, such as `\__consumer_offsets` and `__transaction_state`. The maximum number of topics is 1000 for all Kafka instances.
+
+* `kafka_topic:kafka_topic_partitions:sum`: Number of partitions across all topics in the cluster. This does not include partitions from internal Kafka topics, such as `\__consumer_offsets` and `__transaction_state`.  +
+Your chosen Kafka instance type determines the maximum number of partitions:
+** 100 for a trial instance
+** 1500 for a production instance with a size of 1 streaming unit
+** 3000 for a production instance with a size of 2 streaming units
 --
 
 Broker metrics::
 +
 --
-* `kafka_broker_quota_softlimitbytes`: Maximum amount of storage, in bytes, for this broker before producers are throttled. When this limit is reached, the broker starts throttling producers to prevent them from sending additional data. The maximum storage in the broker is 1000 GiB.
+* `kafka_broker_quota_softlimitbytes`: Maximum amount of storage, in bytes, for this broker before producers are throttled. When this limit is reached, the broker starts throttling producers to prevent them from sending additional data.  +
+Your chosen Kafka instance type determines the maximum storage in the broker:
+** 10 GiB for a trial instance
+** 1000 GiB for a production instance with a size of 1 streaming unit
+** 2000 GiB for a production instance with a size of 2 streaming units
+
 
 * `kafka_broker_quota_totalstorageusedbytes`: Amount of storage, in bytes, that is currently used by partitions in the broker. The storage usage depends on the number and retention configurations of the partitions. This metric must stay below the `kafka_broker_quota_softlimitbytes` metric setting.
 
-* `kafka_controller_kafkacontroller_global_partition_count`: Number of partitions in the cluster. Only the broker that is the current controller in the cluster reports this metric. Any other brokers report `0`. This count includes partitions from internal Kafka topics, such as `\__consumer_offsets` and `__transaction_state`. The maximum number of partitions is 1000. This metric is similar to the `kafka_topic:kafka_topic_partitions:sum` cluster metric.
+* `kafka_controller_kafkacontroller_global_partition_count`: Number of partitions in the cluster. Only the broker that is the current controller in the cluster reports this metric. Any other brokers report `0`. This count includes partitions from internal Kafka topics, such as `\__consumer_offsets` and `__transaction_state`. +
+Your chosen Kafka instance type determines the maximum storage in the broker:
+** 100 for a trial instance
+** 1500 for a production instance with a size of 1 streaming unit
+** 3000 for a production instance with a size of 2 streaming units  +
+This metric is similar to the `kafka_topic:kafka_topic_partitions:sum` cluster metric.
```

Comment:
I suggest keeping this line as part of the metric description before the instance type stuff, where it was before. It makes more sense to me as part of the description itself. It's also more consistent with how we do the same with another metric where we mention something similar. If you were to keep it, you'd still need to add a line space before it in the format I mentioned in my other comment, but best imo to run it with the description.